LLama/Native/NativeApi.LLava.cs

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace LLama.Native;
+
+using clip_ctx = IntPtr;
+public static unsafe partial class NativeApi
+{
+    
+    /*/// <summary>
+    /// Clip Vision Parameters
+    /// </summary>
+    [StructLayout(LayoutKind.Sequential)]
+    public struct clip_vision_hparams
+    {
+        public Int32 image_size;
+        public Int32 patch_size;
+        public Int32 hidden_size;
+        public Int32 n_intermediate;
+        public Int32 projection_dim;
+        public Int32 n_head;
+        public Int32 n_layer;
+        public float eps;
+    };*/
+
+    /// <summary>
+    /// LLaVa Image embeddings 
+    /// </summary>
+    [StructLayout(LayoutKind.Sequential)]
+    public struct llava_image_embed
```

Comment:
I think it's best to to move these out so they're not nested inside `NativeApi`. That class is already very overloaded!